borg/archiver.py

```diff
@@ -513,7 +513,7 @@ def compare_mode(item1, item2):
             if item1[b'mode'] != item2[b'mode']:
                 return '[{} -> {}]'.format(get_mode(item1), get_mode(item2))
 
-        def compare_items(path, item1, item2, hardlink_masters, deleted=False):
+        def compare_items(items, path, item1, item2, hardlink_masters, deleted=False):
```

Comment:
about names:

one would somehow expect, that items are a list of ... items - and item1 is one item and item2 is another item. but in fact, items has a different type of elements - it is a list of output data, so maybe rather call it output?